templates/cli/lib/commands/push.js.twig

```diff
@@ -645,9 +679,92 @@ const createAttribute = async (databaseId, collectionId, attribute) => {
     }
 }
 
-const pushCollection = async ({ all, yes } = {}) => {
-    let response = {};
+const deleteAttribute = async (collection, attribute) => {
+    log(`Deleting attribute ${attribute.key} of ${collection.name} ( ${collection['$id']} )`);
+
+    await databasesDeleteAttribute({
+        databaseId: collection['databaseId'],
+        collectionId: collection['$id'],
+        key: attribute.key,
+        parseOutput: false
+    });
+}
+
+const deepSimilar = (remote, local, collection) => {
+    if (local === undefined) {
+        return undefined;
+    }
+
+    const key = `${chalk.yellow(local.key)} in ${collection.name} (${collection['$id']})`;
 
+    if (remote.type !== local.type) {
```

Comment:
Hmm if we add something new in future, we will surely forget to add it here.
Can we make JSON comparison instead? And if needed we can ignore some keys like $id,name,key.. If relevant.